example/btrfs-subvolumes.nix

```diff
@@ -27,11 +27,21 @@
             end = "100%";
             content = {
               type = "btrfs";
-              mountpoint = "/";
-              subvolumes = [
-                "/home"
-                "/test"
-              ];
+              extraArgs = "-f"; # Override existing partition
```

Comment:
what is this for?

Reply:
No reason, it was in the example from #33 which I copied verbatim. I can remove it (but I would keep the extraArgs option definition)

Reply:
well its good to keep as an example. just thought there could be some idempodency problems without it.